OpenRA.Platforms.Default/FreeTypeFont.cs

```diff
@@ -99,7 +99,7 @@ public void SetSize(int size, float deviceScale)
 			var ascenderValue = (int)ascender >> 6;
 			var descenderValue = (int)descender >> 6;
 
-			Height = (int)((ascenderValue - Math.Abs(descenderValue)) / deviceScale);
+			Height = (int)((ascenderValue - Math.Abs(descenderValue) + 1) / deviceScale);
```

Comment:
Then we would have to do something like this

Reply:
Wouldn't `Height = (int)Math.Ceil((ascenderValue - Math.Abs(descenderValue)) / deviceScale);` or `Height = (int)Math.Round((ascenderValue - Math.Abs(descenderValue)) / deviceScale);` make more sense?